language-support/scala/codegen/src/main/scala/com/digitalasset/codegen/dependencygraph/DependencyGraph.scala

```diff
@@ -36,7 +36,7 @@ private final case class LFDependencyGraph(private val util: lf.LFUtil)
             symmGenTypeDependencies(typeDecl),
             collectDepError = false))
     }
-    val templateNodes = decls.collect {
+    val templateNodes = decls.toStream.collect {
```

Comment:
Did you observe any value from these `toStream` calls? If not, I suggest reverting.

Reply:
looks like it decreased the number of the SOE from 1 out 5 to 1 out 20.